### PR TITLE
Support for processes

### PR DESCRIPTION
**Pull request by neworderofjamie**

### CODE REVIEW

File: nengo_spinnaker/operators/value_source.py [hunk 1]
```diff
@@ -38,9 +41,11 @@ def make_vertices(self, model, n_steps):
         ts = np.arange(max_n) * model.dt
         if callable(self.function):
             values = np.array([self.function(t) for t in ts])
+        elif isinstance(self.function, Process):
+            values = self.function.run_steps(max_n, d=self.size_out, dt=model.dt)
         else:
             values = np.array([self.function for t in ts])
-
+
```
Comment (mundya) [inline nengo_spinnaker/operators/value_source.py]: flake8... your favourite!

File: nengo_spinnaker/builder/ensemble.py [hunk 2]
```diff
@@ -34,6 +35,7 @@ def get_ensemble_sink(model, connection):
 
     if (isinstance(connection.pre_obj, nengo.Node) and
             not callable(connection.pre_obj.output) and
+            not isinstance(connection.pre_obj.output, Process) and
```
Comment (mundya) [inline nengo_spinnaker/builder/ensemble.py:38]: Could you add a test for this?

File: nengo_spinnaker/builder/node.py [hunk 3]
```diff
@@ -111,6 +111,7 @@ def build_node(self, model, node):
             # it.
             vs = ValueSource(
                 node.output,
+                node.size_out,
```
Comment (mundya) [inline nengo_spinnaker/builder/node.py:114]: Also check that this change doesn't break tests.

**Comment by mundya** [comment]
Looks like the failure is because we now technically support Nengo from Git rather than Nengo 2.0.1... I'll modify Travis to account for this.

**Comment by mundya** [comment]
Otherwise, LGTM!

File: nengo_spinnaker/builder/node.py (hunk 3 above)
Comment (neworderofjamie) [inline nengo_spinnaker/builder/node.py:114]: That's the only place that constructor's called!

File: nengo_spinnaker/builder/ensemble.py (hunk 2 above)
Comment (neworderofjamie) [inline nengo_spinnaker/builder/ensemble.py:38]: What do you think of the test I just added? It definitely provides coverage of that code path, probably not so great at showing the input going into the communication channel is anything like white noise though...

File: nengo_spinnaker/builder/node.py (hunk 3 above)
Comment (mundya) [inline nengo_spinnaker/builder/node.py:114]: Fair enough, thanks

File: nengo_spinnaker/builder/ensemble.py (hunk 2 above)
Comment (mundya) [inline nengo_spinnaker/builder/ensemble.py:38]: I've added a test in `test/builder/test_ensemble.py` which tests just this line.  Other than that tests seem good, thank you.

**Comment by mundya** [comment]
Testing against Nengo code from Git makes me very angry! :rage1:

**Comment by mundya** [comment]
@neworderofjamie I'm happy to merge this if you are.

**Comment by neworderofjamie** [comment]
Yey! first submission!

**Pull request merged**